master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/index.md

```diff
@@ -42,7 +41,7 @@ You must have a cluster which meets the following requirements:
 
 This document describes three installation options for {{site.prodname}} using Kubernetes API as the datastore:
 
-1. {{site.prodname}} policy with {{site.prodname}} networking (beta)
+1. {{site.prodname}} policy with {{site.prodname}} networking
```

Comment:
Thanks, I was wondering if it was really still beta!

Reply:
Ugh, looks like I had the wrong impression, we're still missing some features so I need to change this back